react/utils/auditEvents/index.ts

```diff
@@ -0,0 +1,27 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export const createEventObject = (
+  operation: string,
+  entityName: string,
+  subjectId?: any
+) => {
+  const event = {
+    id: uuidv4(),
+    date: new Date().toISOString(),
+    mainAccountName: (window as any).__RUNTIME__.account,
```

Comment:
Instead of accessing the `window` object, you can use the `useRuntime()` hook from `vtex.render-runtime` here. That's better because this `window.__RUNTIME__` object is static, it doesn't get updated after the first page load. The information from `useRuntime()` is dynamic.